common/test/acceptance/pages/lms/fields.py

```diff
@@ -163,26 +163,19 @@ def value_for_text_field(self, field_id, value=None, press_enter=True):
                 query.results[0].send_keys(u'\ue007')  # Press Enter
         return query.attrs('value')[0]
 
-    def value_for_textarea_field(self, field_id, value=None):
+    def set_value_for_textarea_field(self, field_id, value):
         """
-        Get or set the value of a textarea field.
+        Set the value of a textarea field.
         """
         self.wait_for_field(field_id)
-
         self.make_field_editable(field_id)
 
         query = self.q(css='.u-field-{} textarea'.format(field_id))
         if not query.present:
-            return None
-
-        if value is not None:
-            query.fill(value)
-            query.results[0].send_keys(u'\ue004')  # Focus Out using TAB
+            return
```

Comment:
Is it up to the caller to verify that it was set properly?  Is it a standard to quietly fail?

I get that you just dropped the return value, and didn't add this if, but just curious.